lib/backend/k8s/k8s_test.go

```diff
@@ -1967,6 +1987,154 @@ var _ = testutils.E2eDatastoreDescribe("Test Watch support", testutils.Datastore
 		ctx = context.Background()
 	})
 
+	It("Should support watching Profiles", func() {
+		By("Watching a single Profile", func() {
+			name := "default"
```

Comment:
I think we should have two tests here:
- Watch a single profile backed by a Namespace (`name := kns.default`)
- Watch a single profile backed by a serviceaccount (`name := ksa.namespace-2.default`)